templates/template.epp

```diff
@@ -23,7 +23,7 @@ template (name="<%= $template_name %>" type="<%= $type %>"<%-%>
 {
 <% $list_descriptions.each | $element | { -%>
   <%- $element.each |$key, $params| { -%>
-    <%= $key %>(<% $params.each |$p,$v| { -%><%= $p%>="<%= $v %>" <% }%><% } %>)
+    <%= $key %>(<% $params.each |$p,$v| { -%> <%= $p%>="<%= $v %>" <% }%>)<% } %>
```

Comment:
Thanks for the PR! Would it be possible that you provide some tests? The template is a bit challenging to read.

Reply:
updated, i also cleaned up the spec test regex a bit as it seemed a bit overkill and if been picky a bit inefficient (i.e. using *).  let me know if you would prefer me to put https://github.com/voxpupuli/puppet-rsyslog/pull/253/commits/530eeba2f43b3e1e21f1ebe7fc6c96fe90a1d9a7 in a seperate PR